src/components/header/Header.tsx

```diff
@@ -72,7 +74,7 @@ const Header = (): JSX.Element => {
                 'medium-screen:vads-u-font-size--2xl',
```

Comment:
```suggestion
                'medium-screen:vads-u-font-size--xl',
                'small-desktop-screen:vads-u-font-size--2xl',
```

my bad, my recommendation earlier was unnecessarily vague and not very helpful, sorry about that. here's what I would suggest might be more helpful. I'm also fine if there is some wrapping; it's more important that everything be large enough to read than that the wrapping be pixel-perfect. sorry if the first comment was misleading.